### ISSUE TITLE
Docs: Language chosen only affects Gogs aspect of workshop

### ISSUE THREAD

**Issue by benemon** [osevg/workshopper-content#190]
Having recently run this, one comment was that it wasn't immediately obvious that running (for example) the node.js workshop would also require the user to pull down and spin up the Parksmap application - which is obviously a Spring Boot app.

User was briefly concerned that they'd selected the wrong workshop. Maybe add some wording to the Workshop Summary /  Architecture Overview that highlights this?

**Comment by blues-man** [comment]
Absolutely, thanks for your feedback. The idea was to just deploy a container image, which is language-agnostic, but under the hood it's a Spring boot app. Maybe some wording about it or a general purpose icon would help in this ambiguity.

**Comment by benemon** [comment]
Yes - and maybe in the Workshop Summary change:

```
Deploying an application from source code in a Git repository.
```

to:

```
Deploying a <chosen_framework> application from source code in a Git repository.
```

To keep some clarity around the role a user's chosen framework plays here.

**Comment by blues-man** [comment]
I addressed this in this commit
https://github.com/openshift-labs/starter-guides/commit/96abc3ad72a7e6256feff08f7384de07317cd12f

for the second comment, I haven't yet since that requires a creation of multiple files just for this change, we'll keep in consideration.

Thanks for your help!
